client.go

```diff
@@ -149,6 +150,10 @@ type AdminNamespace struct {
 	h jsonrpc.Handler
 }
 
+func (ns AdminNamespace) Handler() jsonrpc.Handler {
+	return ns.h
+}
```

Comment:
Should we do this for all namespaces for consistency?

Reply:
Sure! Why not.